### PR TITLE
[res] strided slice PyTorch example

### PR DESCRIPTION
**Pull request by binarman**
This PR adds strided slice PyTorch example.

Related issue: #7939 

ONE-DCO-1.0-Signed-off-by: Alexander Efimov <a.efimov@samsung.com>

### CODE REVIEW

File: res/PyTorchExamples/examples/strided_slice/__init__.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    def __init__(self, begin, end, stride):
+        super().__init__()
+        self.key = [slice(begin[i], end[i], stride[i]) for i in range(len(begin))]
+
+    def forward(self, input):
+        # this is general way to do input[:, :, 1:5:2, 0:5:2]
+        return input[self.key]
+
+    def onnx_opset_version(self):
+        return 10
```
Comment (binarman) [inline res/PyTorchExamples/examples/strided_slice/__init__.py:16]: opset version 9 does not support strides, so we need at least 10-th version.

Used in #8000

**Review by seanshpark — DISMISSED** [review]
LGTM

**Review by seanshpark — APPROVED** [review]
LGTM

**Comment by binarman** [comment]
@nnfw-bot test nncc-release

**Pull request merged**